public/app/plugins/datasource/testdata/QueryEditor.tsx

```diff
@@ -278,9 +278,10 @@ export const QueryEditor = ({ query, datasource, onChange, onRunQuery }: Props)
         )}
       </InlineFieldRow>
 
-      {scenarioId === TestDataQueryType.RandomWalk && (
-        <RandomWalkEditor onChange={onInputChange} query={query} ds={datasource} />
-      )}
+      {scenarioId === TestDataQueryType.RandomWalk ||
```

Comment:
This one looks kinda weird. Would change it to a ternary operator for readability.